### PR TITLE
Add asset pagination to the bottom of the page

### PR DESCRIPTION
**Pull request by chriscn**
Asset bar should be at bottom of page in line with rest of project

### CODE REVIEW

File: html/common/templates/assets/assetsShopView.twig [hunk 1]
```diff
@@ -38,7 +38,7 @@
                                 <div class="{% if asset.thumbnail|length > 1 %}slick{% endif %}" style="text-align:center;height:200px;">
                                     {% for thumb in asset.thumbnail %}
                                         <div>
-                                            <a href="/asset.php?id={{ asset.assetTypes_id }}{{ SEARCH["SETTINGS"]["SHOWARCHIVED"] ? '&showArchived' : ''}}">
+                                            <a href="{{ CONFIG.ROOTURL }}/asset.php?id={{ asset.assetTypes_id }}{{ SEARCH["SETTINGS"]["SHOWARCHIVED"] ? '&showArchived' : ''}}">
```
Comment (Jbithell) [inline html/common/templates/assets/assetsShopView.twig:41]: This causes a 404 on public sites I'm afraid

File: html/common/templates/assets/assetsShopView.twig [hunk 2]
```diff
@@ -257,6 +257,30 @@
             {% endfor %}
         </div>
     </div>
+    <div class="card-footer">
```
Comment (Jbithell) [inline html/common/templates/assets/assetsShopView.twig:260]: Should this be card-header?

**Review by Jbithell — CHANGES_REQUESTED** [review]

File: html/common/templates/assets/assetsShopView.twig (hunk 1 above)
Comment (chriscn) [reply html/common/templates/assets/assetsShopView.twig]: Resolved in 98f45e00

File: html/common/templates/assets/assetsShopView.twig (hunk 2 above)
Comment (chriscn) [reply html/common/templates/assets/assetsShopView.twig]: Fixed in 6e5622aa

**Comment by chriscn** [comment]
Screenshot 2022-03-28 at 12 06 02
An example of the new asset viewer

File: html/common/templates/assets/assetsShopView.twig (hunk 2 above)
Comment (Jbithell) [reply html/common/templates/assets/assetsShopView.twig]: Whoops I'm wrong - I thought this was at top of page sorry. Will fix :(

**Review by Jbithell — APPROVED** [review]

**Pull request merged**